src/main/java/uk/gov/hmcts/reform/ccd/documentam/client/dmstore/DocumentStoreClient.java

```diff
@@ -0,0 +1,181 @@
+package uk.gov.hmcts.reform.ccd.documentam.client.dmstore;
+
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.ccd.documentam.ApplicationParams;
+import uk.gov.hmcts.reform.ccd.documentam.apihelper.Constants;
+import uk.gov.hmcts.reform.ccd.documentam.dto.DocumentUploadRequest;
+import uk.gov.hmcts.reform.ccd.documentam.exception.ResourceNotFoundException;
+import uk.gov.hmcts.reform.ccd.documentam.exception.ServiceException;
+import uk.gov.hmcts.reform.ccd.documentam.model.DmTtlRequest;
+import uk.gov.hmcts.reform.ccd.documentam.model.DmUploadResponse;
+import uk.gov.hmcts.reform.ccd.documentam.model.Document;
+import uk.gov.hmcts.reform.ccd.documentam.model.PatchDocumentResponse;
+import uk.gov.hmcts.reform.ccd.documentam.model.UpdateDocumentsCommand;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.springframework.http.HttpMethod.GET;
+import static uk.gov.hmcts.reform.ccd.documentam.apihelper.Constants.DM_DATE_TIME_FORMATTER;
+import static uk.gov.hmcts.reform.ccd.documentam.apihelper.Constants.EXCEPTION_ERROR_ON_DOCUMENT_MESSAGE;
+import static uk.gov.hmcts.reform.ccd.documentam.apihelper.Constants.RESOURCE_NOT_FOUND;
+
+@Named
```

Comment:
same comment as earlier about consistency in using either JSR-330 or Spring annotations